packages/grafana-toolkit/bin/grafana-toolkit.js

```diff
@@ -8,7 +8,7 @@ let includeInternalScripts = false;
 const isLinkedMode = () => {
   // In circleci we are in linked mode. Detect by using the circle working directory,
   // rather than the present working directory.
-  const pwd = process.env.CIRCLE_WORKING_DIRECTORY || process.env.PWD;
+  const pwd = process.env.CIRCLE_WORKING_DIRECTORY || process.cwd();
```

Comment:
Think it should be process.env.CIRCLE_WORKING_DIRECTORY || process.env.PWD || process.cwd();

Reply:
I might argue that process.cwd will be enough, but it will not hurt, so added back PWD :)